src/null_driver.py

```diff
@@ -15,16 +15,26 @@ class NullDriver(generic_driver.Driver):
 
     def __init__(self):
         self.fd = -1
-        self.loc = 0L
+        self.loc = 0
         self._rate = 0
         self._last_rate = 0
         self_start_time = None
         self._active_time = 0
         self._total_time = 0
-        self._bytes_transferred = 0L
+        self._bytes_transferred = 0
         self.verbose = 0
         
     def open(self, device=None, mode=None, retry_count=10):
+        """open the device
+            Args:
+                device(str): device name (optional)
```

Comment:
IIRTC `device` is overwritten no matter what, right? We should rename this argument to `_` and note it is unused.

Reply:
I changed (optional) to (not used, but determined by mode), but opted to keep the var name as 'device' .